### ISSUE TITLE
Using COMPOSE_UP_OPTS results in compose error

### ISSUE THREAD

**Issue by nicolaibaralmueller** [ansible/awx#10837]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](http://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I am not entitled to status updates or other assurances.

### Summary

Installing AWX via docker results in the following error: 

`make docker-compose COMPOSE_UP_OPTS=-d`

```
docker-compose -f tools/docker-compose/_sources/docker-compose.yml -d up
Define and run multi-container applications with Docker.

Usage:
  docker-compose [-f <arg>...] [--profile <name>...] [options] [--] [COMMAND] [ARGS...]
  docker-compose -h|--help

Options:
  -f, --file FILE             Specify an alternate compose file
                              (default: docker-compose.yml)
  -p, --project-name NAME     Specify an alternate project name
                              (default: directory name)
  --profile NAME              Specify a profile to enable
  -c, --context NAME          Specify a context name
  --verbose                   Show more output
  --log-level LEVEL           Set log level (DEBUG, INFO, WARNING, ERROR, CRITICAL)
  --ansi (never|always|auto)  Control when to print ANSI control characters
  --no-ansi                   Do not print ANSI control characters (DEPRECATED)
  -v, --version               Print version and exit
  -H, --host HOST             Daemon socket to connect to

  --tls                       Use TLS; implied by --tlsverify
  --tlscacert CA_PATH         Trust certs signed only by this CA
  --tlscert CLIENT_CERT_PATH  Path to TLS certificate file
  --tlskey TLS_KEY_PATH       Path to TLS key file
  --tlsverify                 Use TLS and verify the remote
  --skip-hostname-check       Don't check the daemon's hostname against the
                              name specified in the client certificate
  --project-directory PATH    Specify an alternate working directory
                              (default: the path of the Compose file)
  --compatibility             If set, Compose will attempt to convert keys
                              in v3 files to their non-Swarm equivalent (DEPRECATED)
  --env-file PATH             Specify an alternate environment file

Commands:
  build              Build or rebuild services
  config             Validate and view the Compose file
  create             Create services
  down               Stop and remove resources
  events             Receive real time events from containers
  exec               Execute a command in a running container
  help               Get help on a command
  images             List images
  kill               Kill containers
  logs               View output from containers
  pause              Pause services
  port               Print the public port for a port binding
  ps                 List containers
  pull               Pull service images
  push               Push service images
  restart            Restart services
  rm                 Remove stopped containers
  run                Run a one-off command
  scale              Set number of containers for a service
  start              Start services
  stop               Stop services
  top                Display the running processes
  unpause            Unpause services
  up                 Create and start containers
  version            Show version information and quit
make: *** [Makefile:478: docker-compose] Error 1

```

### AWX version

devel

### Installation method

docker on linux

### Modifications

no

### Operating system

Centos

### Steps to reproduce

`make docker-compose COMPOSE_UP_OPTS=-d`

### Expected results

`docker-compose -f tools/docker-compose/_sources/docker-compose.yml up -d`

### Actual results

`docker-compose -f tools/docker-compose/_sources/docker-compose.yml -d up`

This results in the error as -d should be after up.

**Comment by nicolaibaralmueller** [comment]
Fixed by manually correcting the Makefile

**Comment by life5ign** [comment]
I can confirm this bug in a Python 3.9.1 pyenv virtualenv running on Ubuntu 20.04, with `pip install docker docker-compose`. The last two arguments in the docker-compose command are reversed.

**Comment by gruunday** [comment]
This was already fixed by #10481
